themes/src/main/resources/theme/base/login/resources/js/menu-button-links.js

```diff
@@ -0,0 +1,308 @@
+/*
+ *   This content is licensed according to the W3C Software License at
+ *   https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document
+ *
+ *   File:   menu-button-links.js
+ *
+ *   Desc:   Creates a menu button that opens a menu of links
+ * 
+ *   Modified: Peter Keuter. The original source file is transpiled to ES5
+ */
+
+function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
+function _defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, _toPropertyKey(descriptor.key), descriptor); } }
+function _createClass(Constructor, protoProps, staticProps) { if (protoProps) _defineProperties(Constructor.prototype, protoProps); if (staticProps) _defineProperties(Constructor, staticProps); Object.defineProperty(Constructor, "prototype", { writable: false }); return Constructor; }
+function _toPropertyKey(t) { var i = _toPrimitive(t, "string"); return "symbol" == typeof i ? i : String(i); }
+function _toPrimitive(t, r) { if ("object" != typeof t || !t) return t; var e = t[Symbol.toPrimitive]; if (void 0 !== e) { var i = e.call(t, r || "default"); if ("object" != typeof i) return i; throw new TypeError("@@toPrimitive must return a primitive value."); } return ("string" === r ? String : Number)(t); }
+var MenuButtonLinks = /*#__PURE__*/function () {
```

Comment:
This code looks downcompiled to ES5 from some kind of source, I'd rather not check this in. If you need modern JS syntax you can use the latest features available in evergreen browsers.

Reply:
For context see the `Base IT` action that had a problem with the defined `const`: https://github.com/keycloak/keycloak/actions/runs/7986432593/job/21807156724#step:4:52140
``` 
Run 1: RealmTest.deleteSession » WebDriver com.gargoylesoftware.htmlunit.ScriptException: missing ; before statement (https://localhost:8543/auth/resources/em9ks/login/keycloak/js/menu-button-links.js#12)
```
I would (very) gladly use more modern JS, but `HtmlUnitDriver` used in the Keycloak integration tests don't support this (yet).

Reply:
Yeah, this is a problem we're working on. The test suite will be modernized in the near future to address this. For now you can load your script using `type="module"`, which the test suite will ignore.

Reply:
Ah, that is a nice thing to know. I will update to the ES6 version in that case! Thanks